src/main/java/Duke.java

```diff
@@ -1,10 +1,158 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    private static int taskCounter = 0;
```

Comment:
Consider using the size() method of the ArrayList instead of maintaining another variable to keep track of the number of tasks.

Reply:
This is really helpful to reduce redundant variables. :)